plugins/modules/inventory_workflow_manager.py

```diff
@@ -1810,14 +1810,27 @@ def provision_wired_device_v1(self, device_ip, site_name, device_type):
         try:
             response = self.dnac._exec(family="sda", function='provision_wired_device', op_modifies=True, params=provision_params)
             self.log("Received API response from 'provision_wired_device': {0}".format(response), "DEBUG")
+
             if response:
-                validation_string = "successfully"
-                self.check_task_response_status(response, validation_string, 'provision_wired_device')
-                self.deleted_devices.append(device_ip)
+                exec_id = response.get("executionId")
+                response = self.get_execution_details(exec_id)
+                while True:
+                    if response.get("status") == "SUCCESS":
```

Comment:
we can avoid calling response.get("status") multiple times by storing it in a variable.

Reply:
have used a existing function from the DNAC.py